.flake8

```diff
@@ -5,7 +5,30 @@ import-order-style = pycharm
 max-line-length = 127
 max-complexity = 10
 
-select=E9,E712,F63,F7,F82,W605,T001,T002,T003,T004
+ignore=
+    C901,
```

Comment:
I had to ignore this as we weren't running it before and we now have functions with `max-complexity` ~100